.pre-commit-config.yaml

```diff
@@ -2,6 +2,10 @@ repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     rev: v0.3.4
     hooks:
+      - id: ruff
+        name: Run Ruff on Doc/
+        args: [--exit-non-zero-on-fix]
+        files: ^Doc/
```

Comment:
I think this pre-commit hook only does linting? I think you might need to list a `ruff-format` hook separately if you want the formatting to also be checked via pre-commit

Reply:
Oops!